src/main/java/org/logstash/input/DeadLetterQueueInputPlugin.java

```diff
@@ -34,7 +34,7 @@
 import java.util.function.Consumer;
 
 
-public class DeadLetterQueueInputPlugin {
+public class DeadLetterQueueInputPlugin implements AutoCloseable{
```

Comment:
minor/style - since close is throws IOException this can implement Closeable instead of AutoClosable, which (imo) is a bit more idiomatic